BlazarUI/app/stylus/components/icon.styl

```diff
@@ -28,8 +28,15 @@
   margin-top -5px
   
 .sidebar-item__building-icon-link > .table-icon-container > span > i
-  width 35px
+  width 38px
+  margin-top -8px
 
 .fa-stop
   font-size .55em
   line-height 2.7
+
+.sidebar-item > * > .octicon-repo
```

Comment:
this is pretty funky. are there multiple levels of `octicon-repo` being used in the sidebar, and you're trying to bypass one? I don't think so right? Can you just do `.sidebar-item .octicon-repo` ?